### ISSUE TITLE
Always log output in `showResults()`

### ISSUE THREAD

**Issue by elmeister** [acro5piano/kysely-migration-cli#9]
Thank you for the helpful tool

Do you mind to change [the output f the `showResults()` function](https://github.com/acro5piano/kysely-migration-cli/blob/742a0835c8a628b2c7d1a02e2f6d539fe27a46b4/src/run.ts#L6-L12) to something like the following snippet, so we'll be able to track errors and passed migrations in build output please?

```typescript
function showResults({ error, results }: MigrationResultSet) {
  if (results) {
    for (const it of results) {
      console.log(`> ${it.status}: ${it.migrationName} (${it.direction})`)
    }
  }
  if (error) {
    console.error(error)
    process.exit(1)
  }
}
```

**Comment by acro5piano** [comment]
@elmeister Thanks for the suggestion. Yes, please create a pull request! The change would be helpful.

**Comment by elmeister** [comment]
[PR created](https://github.com/acro5piano/kysely-migration-cli/pull/10), thank you, and once again - :heart: for the project you made

**Comment by acro5piano** [comment]
Released on v0.2.1. Thanks @elmeister 

https://github.com/acro5piano/kysely-migration-cli/releases/tag/v0.2.1
